src/main/kotlin/com/craftschemers/hub/commands/JoinCommand.kt

```diff
@@ -16,12 +20,31 @@ object JoinCommand : ICommand {
     override val permissionMessage = "Permission message"
     override val permission = "permission"
 
-    override fun onCommand(sender: CommandSender, minigame: Minigame, label: String, args: Array<String>): Boolean {
+    private val plugin = Hub.plugin
+
+    override fun onCommand(sender: CommandSender, minigame: Minigame?, label: String, args: Array<String>): Boolean {
         if (sender !is Player) {
-            sender.sendMessage("${ChatColor.RED}You must be a player to execute this command!")
+            sender.sendErrorMessage("You must be a player to execute this command!")
             return true
         }
-        sender.sendMessage("${ChatColor.GOLD}Joined!")
+
+        if (args.isEmpty() || args.size != 1) {
```

Comment:
This could be simplified to
```suggestion
        if (args.size != 1) {
```